nolooking/docker-compose.yml

```diff
@@ -23,3 +23,5 @@ services:
       DEVICE_DOMAIN_NAME: $DEVICE_DOMAIN_NAME
       APP_PASSWORD: $APP_PASSWORD
       APP_HIDDEN_SERVICE: $APP_HIDDEN_SERVICE
+      TOR_PROXY_IP: $TOR_PROXY_IP
+      TOR_PROXY_PORT: $TOR_PROXY_PORT
```

Comment:
> Question: Do we need to re-state envvars in docker-compose.yml?

Yes; Umbrel exposes a number of environment variables, but it's the developer's choice whether they're needed and you would need to be set/use them with the docker services like you've done above.